File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

float dollar;

int main(void)
{
    // Variables
    int cent, count = 0, money[5], i, tmp;
    
    // Assigning values in array
    money[1] = 25;
    money[2] = 10;
    money[3] = 5; 
    money[4] = 1;
    
    // Taking valid input
    do
    {
        dollar = get_float("Change owed: ");
    }
    while (dollar < 0);
    
    // Rounding cents
    cent = round(dollar * 100);
    
    // Calculating total coins needed
    for (i = 1; i <= 4; i++)
    {
        if (cent >= money[i])
        {
            tmp = cent / money[i];
            count += (cent / money[i]);
            cent -= (tmp * money[i]);
        }
    }
    
    printf("%d\n", count);
    
    
    
}

File: pset6/cash/cash.py
from cs50 import get_float

# Prompting user for valid input
while True:
    dollar = get_float("Change owed: ")
    if dollar >= 0:
        break
    
# Rounding cents to nearest integer
cent = round(dollar * 100)

# List of coins
money = []
money.append(25)
money.append(10)
money.append(5)
money.append(1)

# Iterating through list and counting total coin needed
count = 0
for i in range(4):
    
    if cent >= money[i]:
        tmp = int(cent / money[i])
        count += tmp
        cent -= (tmp * money[i])

# Printing solution   
print(count)